### PR TITLE
Add new release 6.0 flags to spec

### PR DESCRIPTION
**Pull request by clarafu**
depends on concourse/concourse#4721 being merged!

### CODE REVIEW

File: jobs/web/spec [hunk 1]
```diff
@@ -852,11 +852,17 @@ properties:
...
 
-  max_conns:
-    env: CONCOURSE_MAX_CONNS
+  api_max_conns:
+    env: CONCOURSE_API_MAX_CONNS
     description: |
-      The maximum number of open connections for a database connection pool.
-    default: 32
+      The maximum number of open connections for the api connection pool.
+    default: 10
```
Comment (cirocosta) [inline jobs/web/spec]: as mentioned in https://github.com/concourse/concourse-bosh-release/pull/66#pullrequestreview-312315186, if we go without a default here, it'd mean that we could go without having to worry about defaults changing in the binary, and then having to change them at the packaging level too 😁

**Comment by cirocosta** [comment]
Hey @clarafu , I just submitted these two PRs:

- update flags to 6.0 release: https://github.com/concourse/concourse-chart/pull/59
- ignore TSA_TEAM_AUTHORIZED_KEYS_FILE https://github.com/concourse/ci/pull/262

if I got it right, with those two and this one you submitted, we should have all of the configs ready to go :grin:

File: jobs/web/spec (hunk 1 above)
Comment (clarafu) [reply jobs/web/spec]: Good point! I'll change that

**Comment by vito** [comment]
i think this is stale now :thinking: i made all the changes in master already. woops!

**Pull request closed without merging**